app/views/competenciesView.html

```diff
@@ -0,0 +1,54 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% extends "layout_unbranded.html" %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+{% if competencies and competencies.length > 0 %} 
+<h2>{{competencies[0].name}} competencies</h2> 
```

Comment:
Could you look at the way the title is done on the homeView.html page on the Main branch? It was changed to make it look a little cleaner

Reply:
fixed